test/tests/functional/pbs_eligible_time.py

```diff
@@ -105,6 +105,6 @@ def test_job_array(self):
         # These are regexp, so they need to be escaped.
         jid_esp = jid.replace('[', '\[').replace(']', '\]')
         m = jid_esp + ";Accrue type has changed to ineligible_time, "
-        m += "previous accrue type was eligible_time for 2[0-5] secs, "
-        m += "total eligible_time=00:00:2[0-5]"
+        m += "previous accrue type was eligible_time for 2[0-9]|30 secs, "
+        m += "total eligible_time=00:00:2[0-9]|30"
```

Comment:
You need to add parentheses around the `|`, otherwise it will match everything on the left, or `30`. 
```suggestion
        m += "total eligible_time=00:00:(2[0-9]|30)"
```